control_panel_api/tests/test_aws.py

```diff
@@ -0,0 +1,65 @@
+from unittest.case import TestCase
+from unittest.mock import patch
+
+from control_panel_api import aws
+
+
+@patch('boto3.client')
+class AwsTestCase(TestCase):
+    def test_create_bucket(self, mock_client):
+        aws.create_bucket('bucketname', 'eu-west-1')
+        mock_client.return_value.create_bucket.assert_called()
+
+    def test_delete_bucket(self, mock_client):
+        aws.delete_bucket('bucketname')
+        mock_client.return_value.delete_bucket.assert_called()
+
+    def test_put_bucket_logging(self, mock_client):
+        aws.put_bucket_logging('bucketname', 'target_bucket', 'target_prefix')
+        mock_client.return_value.put_bucket_logging.assert_called()
+
+    def test_create_policy_json_encoded(self, mock_client):
+        aws.create_policy('bucketname', {'foo': 'bar'})
+        mock_client.return_value.create_policy.assert_called_with(
```

Comment:
Oh, there we are, this is the way you test a method is called with certain params :)